spec/app/domain/authentication/authn-jwt/input_validation/parse_claim_aliases_spec.rb

```diff
@@ -72,6 +72,89 @@
         end
       end
 
+      context "when input has illegal [ ] characters in claim name" do
+        subject do
+          ::Authentication::AuthnJwt::InputValidation::ParseClaimAliases.new().call(
+            claim_aliases: "a[1]:my_claim"
+          )
+        end
+
+        it "raises an error" do
+          expect { subject }.to raise_error(Errors::Authentication::AuthnJwt::ClaimAliasInvalidClaimFormat)
+        end
+      end
+
+      context "when input has illegal [ ] characters in claim value" do
+        subject do
+          ::Authentication::AuthnJwt::InputValidation::ParseClaimAliases.new().call(
+            claim_aliases: "a1:my[1]claim"
+          )
+        end
+
+        it "raises an error" do
+          expect { subject }.to raise_error(Errors::Authentication::AuthnJwt::ClaimAliasInvalidClaimFormat)
+        end
+      end
```

Comment:
Both tests are looking more related to ValidateClaimName class rather to ParseClaimAliases one